package.json

```diff
@@ -81,10 +83,23 @@
     "tabWidth": 2,
     "printWidth": 200
   },
+  "eslintConfig": {
```

Comment:
Just a minor thing: 
Do we need to have this eslintConfig in the package.json anymore?

Reply:
I think that was added for an IDE plugin to work with eslint. Not sure, removed it for now